internal/signalfx-agent/pkg/monitors/haproxy/monitor.go

```diff
@@ -34,6 +34,7 @@ type proxies map[string]bool
 // Config for this monitor
 func (m *Monitor) Configure(conf *Config) (err error) {
 	m.logger = log.WithFields(log.Fields{"monitorType": monitorType, "monitorID": conf.MonitorID})
+	m.logger.Warn("[NOTICE] The haproxy monitor is deprecated and will be removed in a future release. Please use the haproxy receiver instead.")
```

Comment:
I'm not sure about the consistency but to match the metadata.yaml it should be:
```suggestion
	m.logger.Warn("[NOTICE] The haproxy monitor is deprecated and will be removed in a future release. Please use the haproxyreceiver instead.")
```

Reply:
That said it may look like a typo 🤔 ... oh well